File: fall_2020/coding_meetup_01/cats_and_a_mouse/cats.py
# @author Benjamin Alcocer
# https://www.hackerrank.com/challenges/cats-and-a-mouse

n = int(input())

for _ in range(n):
    array = input().replace('\n', '').split()
    z = int(array[2])
    value = abs(z - int(array[0])) - abs(z - int(array[1]))
    print('Mouse C' if valueA == 0 else ('Cat A' if value < 0 else 'Cat B'))

File: 2020_fall/coding_meetup_01/cats_and_a_mouse/cats.cpp
/// @author Benjamin Alcocer
/// https://www.hackerrank.com/challenges/cats-and-a-mouse

#include <iostream>
#include <cmath>

using namespace std;

int main()
{
    int n, x, y, z, value;

    cin >> n;
    for (int i = 0; i < n; i++)
    {
        cin >> x >> y >> z;
        value = abs(z - x) - abs(z - y);
        value == 0 ? cout << "Mouse C" : (value < 0 ? cout << "Cat A" : cout << "Cat B");
        cout << endl;
    }

    return 0;
}
